### PR TITLE
updated inputs and buttons

### PR DESCRIPTION
**Pull request by dlockhart**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3053,7 +3053,7 @@
       "dev": true
     },
     "d2l-icons": {
-      "version": "github:BrightspaceUI/icons#a5d675a7f9862543dd2b60f2cbca2a1d1a61b6f3",
+      "version": "github:BrightspaceUI/icons#615af6ea270ff6206ab8e60b5552d2552707e810",
```
Comment (dlockhart) [inline package-lock.json]: @CarolBusman FYI

File: package-lock.json [hunk 2]
```diff
@@ -3304,7 +3304,7 @@
       "dev": true
     },
     "d2l-organizations": {
-      "version": "github:BrightspaceHypermediaComponents/organizations#f6eedaca763a7ecb82f32fb532b7b36dbe7c0e60",
+      "version": "github:BrightspaceHypermediaComponents/organizations#5b593078fc497fd69213c9f09d77dbe1e873863c",
```
Comment (dlockhart) [inline package-lock.json]: @edwardg8 FYI

File: package-lock.json [hunk 3]
```diff
@@ -3451,7 +3450,7 @@
       }
     },
     "d2l-sequence-navigator": {
-      "version": "github:BrightspaceHypermediaComponents/d2l-sequence-navigator#4643e645a191cbb048c84ec9ed85bde898e14805",
+      "version": "github:BrightspaceHypermediaComponents/d2l-sequence-navigator#e73dd357b25621c1cd26c57e93ac2b3d3197f1ca",
```
Comment (dlockhart) [inline package-lock.json]: @iFun FYI

File: package-lock.json (hunk 2 above)
Comment (edwardg8) [reply package-lock.json]: Cool thanks

**Pull request merged**